### DISCUSSION TITLE
[Bug] `Get-Encoding` returns UTF-8 instead of UTF-16 LE

### DISCUSSION THREAD

**Discussion by nagromc** [ScoopInstaller/Scoop#5894] [Help and Questions]
#### Current Behavior

A little bit of debugging shows that `Get-Encoding` returns UTF-8 for the file of the file https://www.xyplorer.com/download/History.txt

#### Expected Behavior

According to [`chardetect`](https://github.com/chardet/chardet), VS Code, Sublime Text, and Notepad++ the character encoding is UTF-16 LE BOM.

#### Additional context/output

I was trying to write an app manifest for xyplorer (paid version on my personal bucket).

### System details

**Windows version:** 10 22H2

**OS architecture:** 64bit

**PowerShell version:** 5      1      19041  3996   

**Additional software:** [(optional) e.g. ConEmu, Git]

#### Scoop Configuration

```json
{
    "aria2-warning-enabled":  false,
    "last_update":  "2024-02-05T22:29:40.3891902+01:00",
    "scoop_branch":  "master",
    "scoop_repo":  "https://github.com/ScoopInstaller/Scoop"
}
```

**Comment by davidxuang** [comment]
Hi, PowerShell does not have encoding detection. Windows PowerShell defaults to UTF-16LE while PowerShell (Core) defaults to UTF-8.

See: https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_character_encoding?view=powershell-7.4

**Comment by rashil2000** [comment]
Hi @nagromc! What's the error you're facing? I'll try to repro it

**Reply by niheaven** [reply]
Why did you turn so many issues into discussions? Any advantage here? @rashil2000

**Reply by rashil2000** [reply]
If there's something actionable for us to do in Scoop core - implementing a feature or fixing a bug, I keep it as an issue, otherwise help topics, questions, support etc. are moved to Discussions (with the appropriate category - Ideas, Help and Questions etc.).
